superset/sql/parse.py

```diff
@@ -767,6 +781,33 @@ def get_limit_value(self) -> int | None:
 
         return None
 
+    def set_limit_value(self, limit: int) -> None:
```

Comment:
It's been fun implementing the logic in KQL, but I'm considering writing a SIP to get rid of it, since it's not SQL and double the work.